mod/ckeditor/views/default/elgg/ckeditor/set-basepath.js

```diff
@@ -1,2 +1,7 @@
-// This global variable must be set before the editor script loading.
-CKEDITOR_BASEPATH = elgg.config.wwwroot + 'mod/ckeditor/vendors/ckeditor/';
+/**
+ * This view extends elgg.js and sets ckeditor basepath before jquery.ckeditor is loaded
+ */
+require(['elgg'], function(elgg) {
+	// This global variable must be set before the editor script loading.
+	CKEDITOR_BASEPATH = elgg.get_simplecache_url('ckeditor/');
```

Comment:
I'm agreeing it works, but we should document that in the function so we know not to break that functionality in the future.